cmd/airflow.go

```diff
@@ -121,56 +183,7 @@ astro dev run create_user -r Admin -u admin -e admin@example.com -f admin -l use
 		Example:            RunExample,
 		DisableFlagParsing: true,
 	}
-)
-
-func init() {
```

Comment:
Not that I object, but what is the reason for changing how the cli command objects are built?

Reply:
better to avoid `init` and `global vars` as much as we can: two reasons better testing and no magic behavior,